LDAR_Sim/model_code/methods/deployment/stationary_company.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
```

Comment:
Again, I don't think "stationary_company" is a good name - potentially confusing.

I wonder if we should put everything related to scheduling in the same folder or same file called scheduling

Reply:
Same as above